src/targets/intents/index.ejs

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="{{.Locale}}">
+<head>
+  <meta charset="utf-8">
+  <title><%= htmlWebpackPlugin.options.title %></title>
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+  <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16">
+  <link rel="manifest" href="/manifest.json">
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#297EF2">
+  <% if (__TARGET__ !== 'mobile') { %>
+  <link rel="stylesheet" type="text/css" href="//{{.Domain}}/assets/fonts/fonts.css">
+  <% } %>
+  <meta name="theme-color" content="#ffffff">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
+      <link rel="stylesheet" href="<%- file %>">
+  <% }); %>
+  <% _.forEach(htmlWebpackPlugin.files.js, function(file) { %>
+      <script src="<%- file %>" defer></script>
+  <% }); %>
+  <% if (__TARGET__ === 'mobile') { %>
+  <meta name="format-detection" content="telephone=no">
+  <script src="cordova.js" defer></script>
+  <% } else if (__STACK_ASSETS__) { %>
+  {{.CozyClientJS}}
+  {{.CozyBar}}
```

Comment:
You may don't need the cozy-bar here for the intents.

Reply:
I copy/paste too hard, thanks.
I removed the init in the JS file, but forget to remove it here.